frontend/src/components/cluster/ManageClusters.tsx

```diff
@@ -0,0 +1,128 @@
+import { DialogTitle } from '@material-ui/core';
```

Comment:
Since there are some more complex UX decisions we need to make here (like: if we don't allow removing from the default kubeconfig, then users will be puzzled when they don't see certain clusters in the list for removal) and the UI needs to be refined, I have just realized that now that we have a cluster settings view, we could just add a "Remove Cluster" button to the bottom right of that view, which is a much less complex UX and allows us to unblock the "cluster addition" feature for this version.

Reply:
Do you mean move the `Manage Cluster (pen)` Icon next to the settings?